library/goptions.mli

```diff
@@ -150,11 +152,11 @@ val get_ref_table :
 
 (** The first argument is a locality flag.
     [Some true] = "Local", [Some false]="Global". *)
-val set_int_option_value_gen    : bool option -> option_name -> int option -> unit
-val set_bool_option_value_gen   : bool option -> option_name -> bool   -> unit
-val set_string_option_value_gen : bool option -> option_name -> string -> unit
-val set_string_option_append_value_gen : bool option -> option_name -> string -> unit
-val unset_option_value_gen : bool option -> option_name -> unit
+val set_int_option_value_gen    : ?locality:option_locality -> option_name -> int option -> unit
```

Comment:
Why not keep it as a required argument?

Reply:
Well, there is a default value in the type, so it's just a matter of taste.